### ISSUE TITLE
Structs concept has "String" as a pre-requisite, but "String" is unavailable

### ISSUE THREAD

**Issue by dkador** [exercism/rust#1357]
I'm going through the Rust syllabus on exercism. I've successfully completed several, but now am stuck because the next logical concept to approach is structs which requires string, which I can't find anywhere. See screenshot.

![image](https://user-images.githubusercontent.com/77864/136654237-8fe1fd48-1f88-4474-83cf-26f5bb3d660e.png)

I'm happy to provide more details upon request. Apologies if I've missed something obvious. Thanks.

**Comment by coriolinus** [comment]
This is a known issue; it (and other similar missing concept exercises) are why CI has been failing since v3 went public. [example](https://github.com/exercism/rust/runs/3843181440#step:5:10).

Anyone is welcome to open a PR adding a string concept exercise.

**Comment by coriolinus** [comment]
I don't think so. The dependency graph typically reflects the prerequisite concepts for a particular concept exercise, not the prerequisite concepts for the concept in general. If we start deleting prerequisites waiting for implementation, we're likely never to restore all of them.

**Comment by angelikatyborska** [comment]
@coriolinus Seeing how three other identical issues were opened in the last 7 days, I would recommend changing the status of those concept exercises that cannot be yet unlocked from "active" to "wip". This will hide them from users for now.
